universe/Condition.cpp

```diff
@@ -3279,8 +3280,14 @@ std::string Condition::PlanetEnvironment::Description(bool negated/* = false*/)
         }
     }
     std::string species_str;
-    if (m_species_name)
-        species_str = m_species_name->Description();
+    if (m_species_name) {
+        if (ValueRef::ConstantExpr(m_species_name)) {
+            species_str = m_species_name->Eval();
```

Comment:
Could probably use ->Description() here as well; in the case of a ValueRef::Constant[std::string] the result of Eval and Description should, I think, both be just to return m_value. Not a major issue, though is potentially slightly confusing as it imples there's a reason to do Eval in one case and Description in the other.